ocp_resources/forkliftcontroller.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.mtv import MTV
```

Comment:
Please rename the file to match the convention (i.e underscore between words)

Reply:
Done